code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -356,8 +357,8 @@
 //Elite Syndie suit
 /obj/item/clothing/head/helmet/space/hardsuit/syndi/elite
 	name = "elite syndicate hardsuit helmet"
-	desc = "An elite version of the syndicate helmet, with improved armour and fireproofing. It is in EVA mode. Property of Gorlex Marauders."
-	alt_desc = "An elite version of the syndicate helmet, with improved armour and fireproofing. It is in combat mode. Property of Gorlex Marauders."
+	desc = "An military grade hardware which appears to be modified, with improved armour and fireproofing. It is in EVA mode. Property of Gorlex Marauders."
+	alt_desc = "An military grade hardware which appears to be modified, with improved armour and fireproofing. It is in combat mode. Property of Gorlex Marauders."
```

Comment:
My grammar nazi nie pozwala na przecinek przed `when`

Reply:
Cóż za dokładność w sprawdzaniu kodu

Reply:
sobie samemu zmienie potem

Reply:
miałem czas to sobie popatrzyłem

Reply:
![pinkwojak](https://user-images.githubusercontent.com/60329232/101685294-e0a6c380-3a67-11eb-8c37-2c2b6af3dc45.png)
### O NIE CO JA TERAZ ZROBIE